common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -573,7 +573,7 @@ def available_translations(self, transcripts, verify_assets=True):
 
         # If we're not verifying the assets, we just trust our field values
         if not verify_assets:
-            translations = list(other_lang)
+            translations = list(other_lang) if other_lang else []
             if not translations or sub:
                 translations += ['en']
             return set(translations)
```

Comment:
Ah, learn something every day! I'll commit the alternate approach for now and we can bring up the issue of non-English courses.